Makefile

```diff
@@ -144,7 +152,7 @@ test-plugins:
 	pip install haas
 	haas simphony -v
 	haas jyulb -v
-	haas simlammps -v
+	LD_LIBRARY_PATH=./lib/:$LD_LIBRARY_PATH haas simlammps -v
```

Comment:
I tried that way with no luck.

It would rule if lammps.py did something like this: https://github.com/ldx/python-iptables/blob/master/iptc/util.py#L55-64